datatypes/src/raster/raster_properties.rs

```diff
@@ -3,18 +3,23 @@ use crate::util::Result;
 
 use num_traits::FromPrimitive;
 use serde::{Deserialize, Serialize};
+use serde_with::serde_as;
 use std::collections::HashMap;
 use std::convert::TryFrom;
 use std::fmt;
 use std::fmt::{Display, Formatter};
 
 /// This struct stores properties of a raster tile.
 /// This includes the scale and offset of the raster as well as a a description and a map of additional properties.
+#[serde_as]
 #[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Default)]
+#[serde(rename_all = "camelCase")]
 pub struct RasterProperties {
     scale: Option<f64>,
     offset: Option<f64>,
     description: Option<String>,
+    // serialize as a list of tuples because `RasterPropertiesKey` cannot be used as a key in a JSON dict
+    #[serde_as(as = "Vec<(_, _)>")]
```

Comment:
`serda_as` is a feature of the `serde_with` crate wich were are already heavily using in the services crate. This was way easier than implementing a custom serializer and since I wasn't sure if we want to build our cache keys like this, I opted to do it this way for now.